traffic_ops/traffic_ops_golang/user/current.go

```diff
@@ -600,9 +600,9 @@ func ReplaceCurrentV4(w http.ResponseWriter, r *http.Request) {
 	}
 
```

Comment:
Note: Note for Reviewer: `ReplaceCurrentV4` method is used for both Traffic Ops API V4 & V5.